packages/tpetra/core/src/Tpetra_FECrsGraph_decl.hpp

```diff
@@ -207,10 +277,79 @@ namespace Tpetra {
                const Teuchos::RCP<const map_type> & ownedPlusSharedColMap,
                const size_t maxNumEntriesPerRow,
                const Teuchos::RCP<const import_type> & ownedPlusSharedToOwnedimporter = Teuchos::null,
-               const Teuchos::RCP<const map_type> & domainMap = Teuchos::null,
-               const Teuchos::RCP<const map_type> & rangeMap = Teuchos::null,
+               const Teuchos::RCP<const map_type> & ownedDomainMap = Teuchos::null,
+               const Teuchos::RCP<const map_type> & ownedRangeMap = Teuchos::null,
                const Teuchos::RCP<Teuchos::ParameterList>& params = Teuchos::null);
 
+    /// \brief Constructor for locally-indexed assembly specifying a single upper bound for the
```

Comment:
Please add documentation saying why one would use this constructor over the existing constructors.